src/main/java/appeng/parts/storagebus/StorageBusPart.java

```diff
@@ -141,11 +141,13 @@ public StorageBusPart(IPartItem<?> partItem) {
 
     @Override
     protected final void onMainNodeStateChanged(IGridNodeListener.State reason) {
-        var currentActive = this.getMainNode().isActive();
-        if (this.wasActive != currentActive) {
-            this.wasActive = currentActive;
-            this.getHost().markForUpdate();
-            remountStorage();
+        if (reason != IGridNodeListener.State.GRID_BOOT) {
+            var currentActive = this.getMainNode().isActive();
+            if (this.wasActive != currentActive) {
+                this.wasActive = currentActive;
+                this.getHost().markForUpdate();
+                remountStorage();
```

Comment:
Should storage be unmounted when booting status changes?

Reply:
If we unmount storage, would we also unmount drives?

Reply:
Yeah, I think so.

Reply:
It would probably cause unwanted level emitter updates though, wouldn't it?